include/hybridmap/HybridNeutralAtomMapper.hpp

```diff
@@ -0,0 +1,534 @@
+//
+// This file is part of the MQT QMAP library released under the MIT license.
+// See README.md or go to https://github.com/cda-tum/qmap for more information.
+//
+
+#pragma once
+
+#include "Definitions.hpp"
+#include "NeutralAtomLayer.hpp"
+#include "QuantumComputation.hpp"
+#include "cstdint"
+#include "deque"
+#include "fstream"
+#include "hybridmap/HardwareQubits.hpp"
+#include "hybridmap/Mapping.hpp"
+#include "hybridmap/NeutralAtomArchitecture.hpp"
+#include "hybridmap/NeutralAtomDefinitions.hpp"
+#include "hybridmap/NeutralAtomScheduler.hpp"
+#include "hybridmap/NeutralAtomUtils.hpp"
+#include "map"
+#include "operations/Operation.hpp"
+#include "set"
+#include "string"
+#include "utility"
+#include "vector"
+
+#include <cstddef>
+#include <sstream>
+
+namespace qc {
+
+/**
+ * @brief Struct to store the runtime parameters of the mapper.
+ */
+struct MapperParameters {
+  fp       lookaheadWeightSwaps = 0.1;
+  fp       lookaheadWeightMoves = 0.1;
+  fp       decay                = 0.1;
+  fp       shuttlingTimeWeight  = 1;
+  fp       gateWeight           = 1;
+  fp       shuttlingWeight      = 1;
+  uint32_t seed                 = 0;
+};
+
+/**
+ * @brief Helper struct to store the position of a multi qubit gate and the
+ * number of moves needed to execute it.
+ */
+struct MultiQubitMovePos {
```

Comment:
Seems like a utility class to me. Might be worth moving to the Utils file

Reply:
Makes sense